src/group_theory/group_action/basic.lean

```diff
@@ -89,6 +108,24 @@ variables {β}
 lemma exists_smul_eq [is_pretransitive α β] (x y : β) :
   ∃ m : α, m • x = y := is_pretransitive.exists_smul_eq x y
 
+lemma surjective_smul [is_pretransitive α β] (x : β) :
+  surjective (λ c : α, c • x) :=
+exists_smul_eq α x
+
+lemma orbit_eq_univ [is_pretransitive α β] (x : β) :
+  orbit α x = set.univ :=
+(surjective_smul α x).range_eq
+
+/-- The regular action of a group on itself is transitive. -/
+instance regular.is_pretransitive {G : Type*} [group G] : is_pretransitive G G :=
+⟨λ x y, ⟨y * x⁻¹, inv_mul_cancel_right _ _⟩⟩
+
+-- TODO: add an additive version once we have additive opposites
```

Comment:
This comment seems odd; we don't have an additive version of the one on line 120 either, but that doesn't need additive opposites.

Reply:
I moved some code to `defs` so that we have `add_action.is_pretransitive` when we prove properties of multiplicative transitive actions.